### DISCUSSION TITLE
[Bug]kafka组件不能把消费者代码放在两个或以上文件里，只能放在同一个@Consumer装饰的class里

### DISCUSSION THREAD

**Discussion by Yuliang-Lee** [midwayjs/midway#4048] [General / 闲聊]
虽然组件内有查找所有被 `@Consumer` 装饰的模块并且根据内部的 `@KafkaListener ` 执行 `subscribe` 方法，但只有第一次执行的
```
run({
  eachMessage()
})
```
会被注册进消息回调里，这是 `kafkajs` 的逻辑： https://github.com/tulios/kafkajs/blob/master/src/consumer/index.js#L199   ， 多次调用无效

**Comment by waitingsong** [comment]
https://github.com/midwayjs/midway/discussions/4136#discussioncomment-11026460

**Comment by waitingsong** [comment]
https://github.com/midwayjs/midway/pull/4154
